doc/source/report/dynamics.rst

```diff
@@ -3,60 +3,149 @@
+How to get a score for the performance of our simulation?
+The evaluation system is trying to create a score to evaluate the performance of the simulation. Higher the score, better
+the simulation is doing in catching the pattern in the mouse day. Because we have unbalanced data, meaning most of the
+status in one typical mouse day are IS and OTHERS and only a few of them are DRINK and EAT, we should give more weights on
+correct predictions about DRINK and EAT. We first calculate the proportion of the four status within different days we have
+for different mice from different strains and then take average to get the ratio of different status. And then we choose the
+initial weights based on the numbers we get. For example, a mouse day has 21200 timestamps on average. And on average 10000
+of them are IS, 1000 are EAT, 200 are DRINK, and the left 10000 are OTHERS. The ratio is 10000:1000:200:10000 = 1:0.1:0.02:
+0.1. Thus, the weights should be the inverse, 1:10:50:1. 
 
-   - Divide each mouse day into small time intervals, say 5 minutes
-   - For each of the small time intervals, aggregate the data from all mouses in the same strain for all mouse days and estimate the transition probability matrix of a discrete homogeneous Markov Chain model just for this small time interval.
-     - each of these transition probability matrices is estimated by MLE method, where e.g.: 
-     .. math:: P(F_{t+1} | W_{t}) = \frac{N_{WF}}{N_{W.}}
-     where ..math::N_{WF} indicates the counts of transitions from W to F and ..math::N_{W.} indicates the counts of transitions starting from W, no matter where it ends.
-   - Build the whole model by compositing the models for each small time intervals.
+But of course, users can define their own initial weights to fit in their purpose. If the user prefers equal weights, he/she
+can simply set the weight to be (1,1,1,1).
 
-4. Simluation: Use previous estimated model to simluate a typical mouse day for a typical strain. This part could also be used to evaluate the length of the time interval chosen in the previous step.
+The final score is calculated as the number of correct predictions times the weights divided by the number of status. For
+example, if our observed data is “IS IS DRK DRK EAT OTHERS”, and our predicted value based on it is “IS IS IS DRK EAT
+OTHERS”, the score would be (1+1+0+50+10+1)/6 = 10.5. But because our data contains a lot more IS and OTHERS, the score
+ranges from 0 to 1 for our data.
 
-Remarks:
---------
-Testing for each function should be done by the person who wrote the function.
 
-References:
+6. Comparison:
+
+The comparison function plots the simulated behavior dynamics. It plots the the simulated behavior dynamics obtained from the evaluation function for the given strain during the given time period. 
```

Comment:
"The comparison function plots the simulated behavior dynamics obtained from the evaluation function for the given strain during the given time period."